File: lib/src/sdk_api/video_service.dart
import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/video/local_video.dart';
import 'package:dolbyio_comms_sdk_flutter/src/sdk_api/video/remote_video.dart';

/// The VideoService allows changing video settings for the local and remote participants.
///
/// This service is available in SDK 3.7 and later.
/// {@category Services}
class VideoService {
  /// Video settings for the local participant.
  final LocalVideo localVideo = LocalVideo();

  /// Video settings for remote participants.
  final RemoteVideo remoteVideo = RemoteVideo();
}

File: test_app/integration_tests/android_mocks/src/main/java/com/voxeet/sdk/services/VideoService.java
package com.voxeet.sdk.services;

import com.voxeet.sdk.services.video.LocalVideo;
import com.voxeet.sdk.services.video.RemoteVideo;

public class VideoService {
    public LocalVideo local = new LocalVideo();
    public final RemoteVideo remote = new RemoteVideo();
}
